Tests/NIOTests/DatagramChannelTests.swift

```diff
@@ -249,10 +249,7 @@ final class DatagramChannelTests: XCTestCase {
             // will cause EMSGSIZE.
             let bufferSize = 1024 * 5
             var buffer = self.firstChannel.allocator.buffer(capacity: bufferSize)
-            buffer.writeWithUnsafeMutableBytes(minimumWritableBytes: bufferSize) {
-                _ = memset($0.baseAddress!, 4, $0.count)
-                return $0.count
-            }
+            buffer.writeRepeatingByte(4, count: buffer.capacity)
```

Comment:
this isn't a correct translation of the old code. Should be `count: bufferSize`. `ByteBufferAllocator` may over-allocate.